### DISCUSSION TITLE
Running node ace serve --watch on a fresh project results in E_INVALID_FLAG: Unknown flag "watch"

### DISCUSSION THREAD

**Discussion by finestgecko** [adonisjs/core#3132] [Help] [answered]
I just tried to make my first AdonisJS project but when I run `node ace serve --watch` I get this error:

```
joelcieslar@office:~/rest-api$ node ace serve --watch
[ error ]  E_INVALID_FLAG: Unknown flag "watch"
```

Did anyone else run into this problem? I searched online but I couldn't find anything helpful.

## Sample Code (to reproduce the issue)

```
joelcieslar@office:~$ npm init adonis-ts-app@latest rest-api

CUSTOMIZE PROJECT
❯ Select the project structure · api
❯ Enter the project name · rest-api
❯ Setup eslint? (y/N) · true
❯ Setup prettier? (y/N) · true

[ success ]  Project created successfully

joelcieslar@office:~$ cd rest-api

joelcieslar@office:~/rest-api$ node ace serve --watch
[ error ]  E_INVALID_FLAG: Unknown flag "watch"

joelcieslar@office:~/rest-api$ ls
ace       config     node_modules       providers  tsconfig.json
app       contracts  package.json       server.ts
commands  env.ts     package-lock.json  start

joelcieslar@office:~/rest-api$ 
```

## Environment
Ubuntu 21.04

## Package version
` "@adonisjs/core": "^5.3.2"`

This is my first time to use AdonisJS so reinstalling and starting fresh probably won't fix the problem.

## Node.js and npm version
```
joelcieslar@office:~/rest-api$ node -v
v14.17.6
joelcieslar@office:~/rest-api$ npm -v
6.14.15
```

**Comment by thetutlage** [comment, marked as answer]
Hello!

Can you please add this project to a github repo and share its link?

**Reply by finestgecko** [reply]
Thanks for getting back to me. You can find the repo [here](https://github.com/joelcieslar/adonisjs-ace-error). :)

**Reply by finestgecko** [reply]
**Update:** it's working. :tada:

I originally installed Node.js from Snap (v14.17.6). I uninstalled the snap (`sudo snap remove node`) and [reinstalled from nodesource](https://www.tecmint.com/install-nodejs-npm-in-centos-ubuntu/) (also v14.17.6) and it works now.
